src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
```

Comment:
It's not recommended to disable eslint rules without a specific reason. If this rule needs to be disabled, please include a comment explaining why.